src/current/_includes/common/licensing-faqs.md

```diff
@@ -0,0 +1,153 @@
+        1. Select the checkbox to agree to the Terms & Conditions of the [CockroachDB Software License][csl].
+        1. Click the **Generate License Key** button. You will be redirected to the **Enterprise Licenses** page, where you can [start using the key](#set-a-license).
+
+{{site.data.alerts.callout_danger}}
+You will not be able to create more than one **Enterprise Trial** license per day. If you try, the UI will prevent you from proceeding, and the following message will be displayed:
+
+> A new Enterprise Trial license cannot be created at this time because one was recently created. Tell us more about your intended use to find out if you qualify for an Enterprise Free license, or create a new one after October 16, 2024 at 10:49 AM EDT.
+{{site.data.alerts.end}}
+
+{{site.data.alerts.callout_success}}
+For quick local testing of Enterprise features, you can [run a single-node cluster](#single-node-clusters).
+{{site.data.alerts.end}}
+
+## Set a license
+
+{% include common/set-enterprise-license.md %}
+
+## Verify a license
+
+To verify a license, open the [built-in SQL shell]({% link {{ page.version.version }}/cockroach-sql.md %}) and use the [`SHOW CLUSTER SETTING`]({% link {{ page.version.version }}/show-cluster-setting.md %}) command to check the organization name and license key:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+>  SHOW CLUSTER SETTING cluster.organization;
+~~~
+~~~
+  cluster.organization
++----------------------+
+  Acme Company
+(1 row)
+~~~
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+>  SHOW CLUSTER SETTING enterprise.license;
+~~~
+~~~
+             enterprise.license
++-------------------------------------------+
+  crl-0-ChB1x...
+(1 row)
+~~~
+
+The license setting is also logged in the cockroach.log on the node where the command is run:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+$ cat cockroach.log | grep license
+~~~
+~~~
+I171116 18:11:48.279604 1514 sql/event_log.go:102  [client=[::1]:56357,user=root,n1] Event: "set_cluster_setting", target: 0, info: {SettingName:enterprise.license Value:xxxxxxxxxxxx User:root}
+~~~
+
+## Monitor for license expiry
+
+You can monitor the time until your license expires in the following ways:
+
+1. [Prometheus]({% link {{ page.version.version }}/monitor-cockroachdb-with-prometheus.md %}): The `seconds_until_enterprise_license_expiry` metric reports the number of seconds until the license on a cluster expires. It will report `0` if there is no license, and a negative number if the license has already expired. For more information, see [Monitoring and Alerting]({% link {{ page.version.version }}/monitoring-and-alerting.md %}).
+1. [DB console]({% link {{ page.version.version }}/ui-overview.md %}): Several [license expiration messages]({% link {{ page.version.version }}/ui-overview.md %}#license-expiration-message) may be displayed, depending on the status of your cluster's license.
+
+## Renew an expired license
```

Comment:
via @jaiayu :

> for older/current versions: Our new license types (Enterprise Trial and Enterprise Free) will work on older versions, but the expiration behavior will be different. Clusters won't be throttled, but the Enterprise features will immediately stop working at license expiration

@rmloveland make sure you update the FAQ with this, maybe here? maybe throttling section? TBD

Reply:
@jaiayu PS is that also true on a rolling basis going forward?

 eg when v25.1 drops will the "Enterprise Features immediately stop working at license expiration" behavior be applied backwards to e.g. v24.3.X as well as v24.2.X et al
 
or is it a one-time deal with the release of v24.3 and the new licensing scheme

Reply:
I think the response is that it's a one time deal. 

Any binaries that already exist - enterprise features immediately stop working at license expiration
24.3+ and any new binaries of 23.1+ - Throttling behavior will apply.

Reply:
thanks for this callout 

I have added the following info to the 'Monitor for license expiry' section.  It's a bit awkward to describe all this logic in prose, but here's my attempt, PTAL

----

During the transition to the [CockroachDB Software License][csl], expiration behavior (including [throttling](#throttling)) will work as follows depending on your version of CockroachDB.

For versions v23.1.29, v23.2.15, v24.1.7, v24.2.5, and v24.3.0 and later, clusters will behave as described in this documentation.

For versions less than or equal to v23.1.28, v23.2.14, v24.1.6, and v24.2.4, the behavior will be as follows:

- **Enterprise Trial** and **Enterprise Free** licenses will work as expected when you [set a license](#set-a-license).
- However, the expiration behavior will be different. Clusters running these versions won't be [throttled](#throttling); instead, the Enterprise features will immediately stop working at license expiration.